screens/EditProfileScreen.tsx

```diff
@@ -0,0 +1,328 @@
+          console.log("Passt");
+        }
+      })
+      .catch(function (error) {
+        switch (error.response.status) {
+          case 400: {
+            setCasualError(true);
+            break;
+          }
+          case 401: {
+            setNotAuthorized(true);
+            break;
+          }
+          default: {
+            setCasualError(true);
+            break;
+          }
+        }
+      });
+  };
+
+  const handlePasswordChange = () => {
+    axios
+      .patch(`${baseUrl}users/`, {
+        oldPassword: oldPassword,
+        newPassword: newPassword,
+      })
+      .then(function (response) {
+        if (response.status == 200) {
+          setIsPasswordVisible(!isPasswordVisible);
+          setIsChangeSuccess(true);
+        }
+      })
+      .catch(function (error) {
+        switch (error.response.status) {
+          case 400: {
+            setCasualError(true);
+            break;
+          }
+          case 401: {
+            setNotAuthorized(true);
+            break;
+          }
+          case 403: {
+            setOldPassword("");
+            setErrorText("Dein altes Passwort stimmt nicht!");
+            break;
+          }
+          default: {
+            setCasualError(true);
+            break;
+          }
+        }
+      });
+  };
+
+  const onNicknameContentSizeChange = (event: any) => {
+    setNicknameHeight(event.nativeEvent.contentSize.height + 10);
+  };
+
+  const onStatusContentSizeChange = (event: any) => {
+    setStatusHeight(event.nativeEvent.contentSize.height + 10);
+  };
+
+  const handleNicknameChange = (text: string) => {
+    if (text.length <= maxCharactersNickname) {
+      setNickname(text);
+    }
+  };
+
+  const handleStatusChange = (text: string) => {
+    if (text.length <= maxCharactersStatus) {
+      setStatus(text);
+    }
+  };
+
+  const handleNewPassword = () => {
+    setErrorText("");
+    if (confirmNewPassword !== "") {
+      if (newPassword !== confirmNewPassword) {
+        setErrorText("Die Passwörter stimmen nicht überein!");
+      }
+    }
+  };
+
+  const handleConfirmNewPassword = () => {
+    setErrorText("");
+    if (newPassword !== "") {
+      if (newPassword !== confirmNewPassword) {
+        setErrorText("Die Passwörter stimmen nicht überein!");
+      }
+    }
+  };
+
+  useFocusEffect(
+    React.useCallback(() => {
+      setIsPasswordVisible(false);
+      setIsChangeSuccess(false);
+
+      return () => {
+        setIsPasswordVisible(false);
+        setIsChangeSuccess(false);
+        setOldPassword("");
+        setNewPassword("");
+        setConfirmNewPassword("");
+      };
+    }, []),
+  );
+
+  if (notAuthorized) {
+    return (
+      <SafeAreaView className="flex bg-white justify-center items-center">
+        <Text className="text-lg">Du musst dich erst anmelden</Text>
+        {/* <TouchableOpacity
+          style={styles.errorButton}
+          onPress={() => navigation.navigate("Authentification" as never)}
```

Comment:
Warum auf "Authentification" und nicht Log In? Was ist Authentification?

Reply:
In Call geklärt: Authentification ist die seite für Login als auch registrierung